integrated_channels/moodle/models.py

```diff
@@ -82,6 +148,38 @@ class MoodleEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfiguratio
         )
     )
 
+    decrypted_token = EncryptedCharField(
+        max_length=255,
+        verbose_name="Encrypted Webservice Token",
+        blank=True,
+        help_text=_(
+            "The encrypted API user's token used to obtain new tokens."),
+        default="",
+    )
```

Comment:
[nit] Once we address the above feedback, please make sure to add back null=True and remove default value.